p/scripts/main.js

```diff
@@ -464,11 +464,16 @@ function prev_feed() {
 	const feeds = document.querySelectorAll('#aside_feed .feed');
 	for (let i = feeds.length - 1; i >= 0; i--) {
 		const feed = feeds[i];
-		if (found && getComputedStyle(feed).display !== 'none') {
+		if (feed.classList.contains('active')) {
+			found = true;
+			continue;
+		}
+		if (!found) {
+			continue;
+		}
+		if (feed.dataset.unread != 0 && getComputedStyle(feed).display !== 'none') {
```

Comment:
I'm only taking a quick glance, but is this even reachable? The first if statement sets found to true and immediately continues to the next iteration, while the one directly following continues if found isn't true?

Reply:
This is reachable. I've tested it with many different configurations and they all worked.
The first if statement catches the current feed. The second if statement is there to not process feeds until we found the current feed. The third if statement does the job of switching feeds.

I hope this makes sense!